roles/matrix-ma1sd/defaults/main.yml

```diff
@@ -83,7 +83,7 @@ matrix_ma1sd_threepid_medium_email_connectors_smtp_password: ""
 # so that ma1sd can rewrite the original URL to one that would reach the homeserver.
 matrix_ma1sd_dns_overwrite_enabled: false
 matrix_ma1sd_dns_overwrite_homeserver_client_name: "{{ matrix_server_fqn_matrix }}"
-matrix_ma1sd_dns_overwrite_homeserver_client_value: "http://matrix-synapse:8008"
+matrix_ma1sd_dns_overwrite_homeserver_client_value: "http://matrix-synapse:{{ matrix_synapse_container_client_api_port }}"
```

Comment:
We should move this to `group_vars/matrix_servers`. You can define `matrix_ma1sd_dns_overwrite_homeserver_client_value` as empty here and ensure it's set via `roles/matrix-ma1sd/tasks/validate_config.yml `